README.md

```diff
@@ -504,6 +504,8 @@ To handle this the client library can be put in multiprocess mode.
 This comes with a number of limitations:
 
 - Registries can not be used as normal, all instantiated metrics are exported
+  - Registering metrics to a registry later used by a `MultiProcessCollector`
```

Comment:
this is under a section called `(Gunicorn)` however this seems more generic and not tied to Gunicorn, right? I hit this issue with UWSGI.

Reply:
this also begs the question of, what is the case someone does want to register a metric to a registry?

Reply:
The Gunicorn specification is a good point, perhaps `(E.g. Gunicorn)` instead to make it clear that is an example?

There are lots of valid use cases to register a metric when in non-multiprocess mode, it just doesn't work for multi-process right now. For example, when building a custom exporter you might have one registry for internal metrics, and a second registry for the metrics coming from something being probed.

Reply:
roger that